internal/rbd/controllerserver.go

```diff
@@ -833,6 +878,48 @@ func (cs *ControllerServer) DeleteVolume(
 	}
 	defer cs.OperationLocks.ReleaseDeleteLock(volumeID)
 
+	if isMigrationVolHandle(volumeID) {
+		log.DebugLog(ctx, "migration volume handle : %s", volumeID)
+		pMH, pErr := parseMigrationVolHandle(volumeID)
+		if pErr != nil {
+			log.DebugLog(ctx, "failed to parse migration volume handle: %s , err: %v", volumeID, pErr)
+
+			return nil, pErr
+		}
+		rv := &rbdVolume{}
+		if pMH != nil {
+			rv.RbdImageName = pMH.ImageName
+			rv.Monitors = pMH.Mons
+			rv.Pool = pMH.Pool
+		}
+		log.DebugLog(ctx, "filled rbd volume: %+v", rv)
+		monsMap := make(map[string]string)
+		monsMap["monitors"] = rv.Monitors
```

Comment:
why create an extra map just for this variable update `getClusterIDFromMigrationVolume` to take a string input?

Reply:
yeah, was thinking about it.. I dont see a strong reason to put that on map, so let me revert that to string.

Reply:
done.. thanks.. now it take string input.